test/functional/test-cache-sw-core.js

```diff
@@ -243,35 +247,215 @@ runner.run('Cache SW', () => {
     });
   });
 
+  describe('expired', () => {
+    let response;
+    let clock;
+
+    beforeEach(() => {
+      clock = sandbox.useFakeTimers();
+      clock.tick();
+      response = new Response('');
+    });
+
+    describe('with cache-control and date', () => {
+      describe('date in the past', () => {
+        beforeEach(() => {
+          response.headers.set('date', new Date(Date.now() - 1000)
```

Comment:
Not sure I can do this one, since it will start failing in a year. Maybe just a comment?